src/LeanCode.Contracts/NotificationEnvelope.cs

```diff
@@ -13,8 +13,8 @@ public sealed class NotificationEnvelope
     public NotificationEnvelope(Guid id, ITopic topic, object notification)
     {
         Id = id;
-        TopicType = topic.GetType().FullName!;
-        NotificationType = notification.GetType().FullName!;
+        TopicType = topic.GetType().ToString();
```

Comment:
We want `TopicType` to stay as it was, cause this version doesn't include namespace IIRC.